docs/articles/index.md

```diff
@@ -9,9 +9,19 @@ This module provides the next generation Maintenance UI based on React/Typescrip
 As a prerequisite, you need:
 
 - NodeJS (9.5.0)
+- Python (2.7.18)
+- .Net Framework SDK 2.0
```

Comment:
Maybe we should add prerequisite for windows and for linux. The .NET Framework is not required for the web interface.